index.html

```diff
@@ -323,155 +327,186 @@ <h1>Multi-Post Stories Gain+Glory</h1>
-				</div>
-				<div>
-					<h1>
-					    Frameworks	
-					</h1>
-				</div>
+					<!-- about-me-card  2 -->
+					<div class="about-me-cards">
+						<div class="icon-languages">
+							<img src="images/icon-frameworks-2.svg" alt="languages">
+						</div>
+						<div>
+							<h1>
+								Frameworks	
+							</h1>
+						</div>
 						<div class="abt-frameworks">
 							<ul class="abt-languages">
 								<li>
 									<a href="#">
-								<button id="ruby">Bootstrap</button>
-								</a>
+										<button id="ruby">Bootstrap</button>
+									</a>
 								</li>
 								<li>
 									<a href="#">
-								<button id="ruby">Ruby on rails</button>
-								</a>
+										<button id="ruby">Ruby on rails</button>
+									</a>
 								</li>
 								<li>
 									<a href="#">
-								<button id="ruby">RSpec</button>
-								</a>
+										<button id="ruby">RSpec</button>
+									</a>
 								</li>
 								<li>
 									<a href="#">
-								<button id="ruby">Capybara</button>
-								</a>
+										<button id="ruby">Capybara</button>
+									</a>
 								</li>
 								<li>
 									<a href="#">
-								<button id="ruby">Selenium</button>
-								</a>
+										<button id="ruby">Selenium</button>
+									</a>
 								</li>
 							</ul>
 
 						</div>
-			</div>
+					</div>
 
-			<!-- about-me-card  3 -->
-			<div class="about-me-cards">
-				<div class="icon-languages">
-					<img src="images/icon-skills.svg" alt="languages">
-				</div>
-				<div>
-					<h1>
-					   Skills 
-					</h1>
-				</div>
+					<!-- about-me-card  3 -->
+					<div class="about-me-cards">
+						<div class="icon-languages">
+							<img src="images/icon-skills.svg" alt="languages">
+						</div>
+						<div>
+							<h1>
+								Skills 
+							</h1>
+						</div>
 						<div class="abt-frameworks">
 							<ul class="abt-languages">
 								<li>
 									<a href="#">
-								<button id="ruby">Codekit</button>
-								</a>
+										<button id="ruby">Codekit</button>
+									</a>
 								</li>
 								<li>
 									<a href="#">
-								<button id="ruby">GitHub</button>
-								</a>
+										<button id="ruby">GitHub</button>
+									</a>
 								</li>
 								<li>
 									<a href="#">
-								<button id="ruby">Codepen</button>
-								</a>
+										<button id="ruby">Codepen</button>
+									</a>
 								</li>
 								<li>
 									<a href="#">
-								<button id="ruby">GitLab</button>
-								</a>
+										<button id="ruby">GitLab</button>
+									</a>
 								</li>
 								<li>
 									<a href="#">
-								<button id="ruby">Terminal</button>
-								</a>
+										<button id="ruby">Terminal</button>
+									</a>
 								</li>
 							</ul>
 
 						</div>
+					</div>
+
+				</div>
 			</div>
+		</section>
+		<section>
+			<div class="form-section">
+				<div> 
+					<h1 class="frm-txt">I'm always interested in hearing about new projects,
```

Comment:
- To help improve your page's structure and for better accessibility, it is not advisable to have more than one `h1` tag on your page. Please fix this and remember to follow `hx` heirachy 😄. You can refer to this [article](https://www.searchenginejournal.com/on-page-seo/header-tags/#:~:text=HTML%20header%20tags%20follow%20a,further%20structure%20within%20those%20subsections.) for a better understanding.

Reply:
Thank you, I will be changing this.